### ISSUE TITLE
--show-gitignored, but do not enter

### ISSUE THREAD

**Issue by jaklan** [Canop/broot#409]
`broot` allows special paths in the config:
```
special_paths: {
    "/media/slow-backup-disk"             : no-enter
    "/home/dys/useless"                   : hide
    "/home/dys/my-link-I-want-to-explore" : enter
}
```
which are quite useful for dealing e.g. with `.git` folders:
```
special_paths: {
    "**/.git": no-enter
}
```
so they are still visible in `broot` (especially when used as a replaced for `tree`), but don't mess the output with their content.

It would be nice to achieve similar effect with git-ignored directores.

**Comment by Canop** [comment]
Reformulation: It looks to me that the solution would be

* a new mode in which gitignored files are displayed but not automatically entered
*  the verbs toggling this mode

**Comment by faelin** [comment]
@Canop did you ever resolve this issue?

**Comment by faelin** [comment]
I agree with your reformulation, except that only *directories* that are git-ignored should be shown (and not entered), while git-ignored file nodes should remain hidden by default.

**Comment by faelin** [comment]
As a separate (but related) request—which I might move into a separate [feature request] issue—it would be extremely nice to be able to pass special-paths to broot via CLI flags, as in
```
$> broot --hide="/home/dys/useless" --no-enter="/media/slow-backup-disk" --no-enter="**/.git" --enter="/home/dys/git-ignored-path"
```
(not that in my example, flags can be repeated multiple times, but semicolon-separated paths might be acceptable instead)

**Comment by Canop** [comment]
Not sure about the best arg syntax but this feature sounds reasonable and with no downside. Please create a separate issue for it.
